CONTRIBUTING.md

```diff
@@ -60,29 +60,27 @@ git fetch upstream
 git merge upstream/master
 ```
 
-### Step 6 - Install the Python module and the CLI
+### Step 6 - Build and Run
 
-To use and run the source code you just cloned from your fork, you need to install BigchainDB on your computer.
-The core of BigchainDB is a Python module you can install using the standard [Python packaging tools](http://python-packaging-user-guide.readthedocs.org/en/latest/).
-We highly suggest you use `pip` and `virtualenv` to manage your local development.
-If you need more information on how to do that, refer to the *Python Packaging User Guide* to [install `pip`](http://python-packaging-user-guide.readthedocs.org/en/latest/installing/#requirements-for-installing-packages) and to [create your first `virtualenv`](http://python-packaging-user-guide.readthedocs.org/en/latest/installing/#creating-virtual-environments).
+To use and run the source code you just cloned from your fork for local development and testing. Run the following command to build a docker image from your cloned fork
 
-Once you have `pip` installed and (optionally) you are in a virtualenv, go to the root of the repository (i.e. where the `setup.py` file is), and type:
+```text
+docker-compose -f docker-compose.yml build
+```
+
+NOTE: If you already ran the above command once you can just run `docker-compose -f docker-compose.yml build bdb` to speed up the build process for any subsequent builds
```

Comment:
Since the default compose file is `docker-compose.yml` we can avoid the `-f docker-compose.yml` option.